File: 628/maximum-product-of-three-numbers.go
func maximumProduct(nums []int) int {
	length := len(nums)
	for i := 0; i < length-1; i++ {
		for j := i + 1; j < length; j++ {
			if nums[i] > nums[j] {
				nums[i], nums[j] = nums[j], nums[i]
			}
		}
	}

	maxProduct := nums[length-1] * nums[length-2] * nums[length-3]
	if maxProduct < nums[length-1]*nums[0]*nums[1] {
		maxProduct = nums[length-1] * nums[0] * nums[1]
	}
	if maxProduct < nums[0]*nums[1]*nums[2] {
		maxProduct = nums[0] * nums[1] * nums[2]
	}

	return maxProduct
}

File: 628/maximum-product-of-three-numbers.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumProduct = function (nums) {
    nums.sort((a, b) => a - b);
    let len = nums.length;
    let maxProduct = nums[len - 1] * nums[len - 2] * nums[len - 3];
    maxProduct = Math.max(maxProduct, nums[len - 1] * nums[0] * nums[1]);
    maxProduct = Math.max(maxProduct, nums[0] * nums[1] * nums[2]);
    return maxProduct;
}